src/openklant/components/klantinteracties/migrations/0020_alter_digitaaladres_soort_digitaal_adres.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.15 on 2024-10-18 10:38
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("klantinteracties", "0019_alter_partij_indicatie_geheimhouding"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="digitaaladres",
+            name="soort_digitaal_adres",
+            field=models.CharField(
```

Comment:
could you remove the migrations 0020 and 0021 and re-run `makemigrations` to remove this unnecessary migration?